### DISCUSSION TITLE
Nengo, save data each epoch

### DISCUSSION THREAD

**Discussion by cnrgbot** [nengo/nengo-zoo#740] [General Discussion]
> _Archived from the Nengo Discourse forum._ Originally posted by **Piergiuseppe** on 2020-07-20. (Forum topic #1265, post #1 / id 3405)

Hi I was wondering if it is possible to do the following (keras) into a similar manner with Nengo_dl simulator. 
history = model.fit(X, Y, validation_split=0.33, epochs=150, batch_size=10, verbose=0)
plt.plot(history.history['accuracy'])
plt.plot(history.history['val_accuracy'])
My goal is to get the validation accuracies and training accuracies at each epoch.
